README.md

```diff
@@ -19,194 +19,62 @@
-
-Security for the development image is enabled via Open Liberty's `<quickStartSecurity>` configuration. This mechanism provides a basic username/password to the server and is intended for development. It is intentionally removed when an official production image is built.
-
-Log in as the `admin` user to see both the system and application metrics in a text format.   The password for this `admin` user will be generated by the container.  
-
-To get the generated password for your project, you can exec in the container with these steps:
-
-1. List the name of the pod running in the current namespace:
-
-    $ kubectl get pods
-
-1. Use the oc tool to open a remote session to your pod, and print the keystore_password created for the Open Liberty server:
-
-    $ oc rsh <pod_name>
-    $ grep keystore /projects/target/liberty/wlp/usr/servers/defaultServer/server.env
-
-    keystore_password=2r1aquTO3VVUVON7kCDdzno
-
-So in this example the password value would be: `2r1aquTO3VVUVON7kCDdzno`
-
-
-#### Junit 5
-
-The default template uses JUnit 5. You may be used to JUnit 4, but here are some great reasons to make the switch https://developer.ibm.com/dwblog/2017/top-five-reasons-to-use-junit-5-java/
-
-
 ## Getting Started
 
-1. Perform an `oc login` to your Open Shift cluster for development purposes.
+> It's recommended to use the latest version of OpenShift Do (odo). You can install odo using [these instructions](https://odo.dev/docs/installing-odo/)
 
-1.  Install the latest vesion of OpenShift Do (odo) using [these instructions](https://odo.dev/docs/installing-odo/)
+1. Perform an `oc login` to your cluster.
+
+1. Create a new odo project
 
-    <!-- Keep for now in case future changes require experimental mode
-    1. Odo must be run in experimental mode at the 1.x level in order to access the devfile support. Enable experimental mode using the odo CLI:
     ```shell
-    odo preference set experimental true
+    odo project create my-project
     ```
-    --->
 
-1. Create a new folder in your local directory and create an odo project using the odo CLI, e.g.:
+    This will create a new namespace in your cluster called `my-project`
+
+1. Initialize the local folder with the Open Liberty stack
+
     ```shell
     mkdir my-project
     cd my-project
-    odo project create <project namespace name>
-    ```
-    The project namespace name will be used to create an OpenShift namespace that will house your application and related pods
-
-1. Initialize the local folder using the Open Liberty stack.
-    ```shell
     odo create java-openliberty --starter
     ```
 
-    This will initialize an Open Liberty project and download the default starter app artifacts.
+    This will download the default starter app
 
-1. Once your project has been initialized, you can push your application to your OpenShift cluster for the first time using the following command:
+1. Push your app to your cluster
 
     ```shell
     odo push
+    odo watch
```

Comment:
Did you really mean to document this sequence?   

I would usually suggest:
1. odo push
2. odo log -f   
(then wait for the test success message).

Reply:
I don't think the starter has any tests to run.